web/src/js/__tests__/ducks/tutils.js

```diff
@@ -8,4 +14,58 @@ export function createStore(parts) {
     )
 }
 
-export { default as TFlow } from './_tflow'
+export { TFlow }
+
+export function TStore(){
+    let tflow = new TFlow()
+    return mockStore({
+        ui: {
```

Comment:
Any particular reason why we don't use the respective reducers here? This currently is a good candidate where we could get out of sync.